docs/ENTITY_RELATIONSHIP_DIAGRAM.md

```diff
@@ -132,6 +132,9 @@ erDiagram
 	Proposal }o--|| User : "is created by"
 	ProposalVersion }o--|| User : "is created by"
 	BulkUpload }o--|| User : "is created by"
+	User }o--o{ Changemaker : "is granted access within"
```

Comment:
6. This almost says "`Changemaker` in `ChangemakerRole` is the object but it's still not clear.

Reply:
Regarding the name clarity: what do you think of `UserChangemakerRole`

Meaning "the user's role within a changemaker"

(and then the same for the other two)

Reply:
I've made this change, I think it is more clear (unresolve if not!)

Reply:
Between yesterday's review and today I thought perhaps `ChangemakerAccess` or now maybe `UserChangemakerAccess`, `UserChangemakerPermission`, `UserPermissionOnChangemaker` would be better, assuming we're talking about access controls on objects associated with that Changemaker. I think that's what we're talking about but when you say "within a changemaker" above, it makes me worry that we're substituting these ACLs for group membership at the same time. I would say "the user's access permissions on a changemaker's objects." In other words, these are truly User permissions and not Changemaker permissions. The User is the subject and the Changemaker is the object, right? I will re-review and see if that is consistent.

Reply:
I like `UserPermissionOnChangemaker` -- and then in stead of `access_type` it maybe should be `permission`

Instead of `roles` we then have `permissions`

Thanks @bickelj for the refinements!

Reply:
Futher improvement from our call right now: `PermissionOnChangemaker` is even better as it becomes more flexible down the line (we may have this table grant permissions to groups instead of / as well as individual users)

Reply:
We landed on `UserChangemakerPermissions` for now; we can adjust this name later if / when the scope expands beyond individual users.